section2-the-ruby-programming-language/strings/analyzer.rb

```diff
@@ -0,0 +1,13 @@
+puts "Enter your first name"
+first_name = gets.chomp
+
+puts "Enter your last name"
+last_name = gets.chomp
+
+full_name = first_name + " " + last_name
+reversed_name = full_name.reverse
+name_length = first_name.length + last_name.length
```

Comment:
Shouldn’t full name character length be full_name.length?

Reply:
If using `full_name.length` the spacing in between `first_name` and `last_name` will be counted. The homework did specify that the spacing in between would not be counted.

Reply:
Ok